source/_integrations/zwave_js.markdown

```diff
@@ -311,6 +311,32 @@ Value Notification example:
 }
 ```
 
+### Value updated events
+
+Due to some devices not following the Z-Wave spec, there are scenarios where a device will send a value update but a state change won't be detected in Home Assistant. To address the gap, the `zwave_js_value_updated` event can be listened to to capture any value updates that are received by an entity. This event is **enabled on a per device basis.** The following devices currently support this event:
```

Comment:
Should we mention somewhere that these devices will be considered to have assumed state?